### PR TITLE
Update wizard helper texts

### LINKED ISSUE #407: Update helper texts
At the Wizard, every helper text should be updated in order to correspond to the latest changes on the software.

### PR DESCRIPTION
**Pull request by fsoupimenta**
Closes #407

### CODE REVIEW

File: source/store/help_button_text.py [hunk 1]
```diff
@@ -2,23 +2,30 @@
...
                " <br>\u03c9(G) == a(\u2113(G)) <br> \u03bb \u2081(G) > (\u03bb\u2081(c(G)))/2 <br>" \
-           "\u03c9(G) == a(\u2113(G)) OR \u03bb \u2081(G) > (\u03bb\u2081(c(G)))/2 </p>"
+               "\u03c9(G) == a(\u2113(G)) OR \u03bb \u2081(G) > (\u03bb\u2081(c(G)))/2 <br> <br> You can access our " \
+               "<a href=""https://github.com/GraphFilter/GraphFilter/wiki/Dictionary""> dictionary</a> " \
+               "to find out more about each invariant.</p>"
 
 method = "<p> The 'Filter graphs'' will filter a list of graphs and return all those that satisfy the conditions " \
-             "marked in the previous screens. <br> The 'Find Counterexample' will run until it finds a graph that " \
-             "does not meet the conditions, if any. </p>"
+             "marked in the previous screens. <br> The 'Find an example' will run until it finds a first graph that " \
```
Comment (atilaajones) [inline source/store/help_button_text.py:10]: Filter Graphs: The conditions will be marked on the following screens (not previously)

**Review by atilaajones — CHANGES_REQUESTED** [review]

File: source/store/help_button_text.py (hunk 1 above)
Comment (fsoupimenta) [reply source/store/help_button_text.py]: Done!

**Review by LBeghini — APPROVED** [review]

**Review by atilaajones — APPROVED** [review]

**Review by atilaajones — CHANGES_REQUESTED** [review]
@fsoupimenta 
Can you update the final text of review page?

**Suggestion:**
 After the filtering is performed, two files will be generated:'
- A list of graphs (.g6 format) containing all the graphs that meet the filtering conditions. In the case of the 'Find an example' method, only one graph will be returned.
- A text file containing all the filtering input and output information.
The returned graphs will be opened automatically once the filtering process is complete.

            
**What do you think about it?**

**Comment by fsoupimenta** [comment]
> @fsoupimenta Can you update the final text of review page?
> 
> **Suggestion:** After the filtering is performed, two files will be generated:'
> 
> * A list of graphs (.g6 format) containing all the graphs that meet the filtering conditions. In the case of the 'Find an example' method, only one graph will be returned.
> * A text file containing all the filtering input and output information.
>   The returned graphs will be opened automatically once the filtering process is complete.
> 
> **What do you think about it?**

Instead of a "text file" i think that we can use "PDF file". do you agree?

**Comment by atilaajones** [comment]
> > @fsoupimenta Can you update the final text of review page?
> > **Suggestion:** After the filtering is performed, two files will be generated:'
> > 
> > * A list of graphs (.g6 format) containing all the graphs that meet the filtering conditions. In the case of the 'Find an example' method, only one graph will be returned.
> > * A text file containing all the filtering input and output information.
> >   The returned graphs will be opened automatically once the filtering process is complete.
> > 
> > **What do you think about it?**
> 
> @atilaajones, Instead of a "text file", i think that we can use "PDF file". do you agree?

You're right, it's even better.

**Review by atilaajones — APPROVED** [review]

**Pull request merged**